plugins/auth-backend/src/providers/aws-alb/provider.ts

```diff
@@ -94,40 +131,147 @@ export class AwsAlbAuthProvider implements AuthProviderRouteHandlers {
     return Promise.resolve(undefined);
   }
 
+  private async getResult(req: express.Request): Promise<AwsAlbResult> {
+    const jwt = req.header(ALB_JWT_HEADER);
+    const accessToken = req.header(ALB_ACCESSTOKEN_HEADER);
+
+    if (jwt === undefined) {
+      throw new AuthenticationError(
+        `Missing ALB OIDC header: ${ALB_JWT_HEADER}`,
+      );
+    }
+
+    if (accessToken === undefined) {
+      throw new AuthenticationError(
+        `Missing ALB OIDC header: ${ALB_ACCESSTOKEN_HEADER}`,
+      );
+    }
+
+    try {
+      const headers = getJWTHeaders(jwt);
+      const key = await this.getKey(headers.kid);
+      const claims = JWT.verify(jwt, key) as AwsAlbClaims;
+
+      if (this.issuer && claims.iss !== this.issuer) {
+        throw new AuthenticationError('Issuer mismatch on JWT token');
+      }
+
+      const fullProfile: PassportProfile = {
```

Comment:
Not sure about this part. Maybe this should be configurable as well? That would mean a handler for generating the `fullProfile` that is then directly passed to the `AuthHandler`.